loading data... data loaded!
model architecture: CNN-non-static
using: word2vec vectors
[('image shape', 64, 300), ('filter shape', [(100, 1, 3, 300), (100, 1, 4, 300), (100, 1, 5, 300)]), ('hidden_units', [100, 2]), ('dropout', [0.5]), ('batch_size', 50), ('non_static', True), ('learn_decay', 0.95), ('conv_non_linear', 'relu'), ('non_static', True), ('sqr_norm_lim', 9), ('shuffle_batch', True)]
Traceback (most recent call last):
  File "conv_net_sentence.py", line 325, in <module>
    dropout_rate=[0.5])
  File "conv_net_sentence.py", line 85, in train_conv_net
    layer0_input = Words[T.cast(x.flatten(),dtype="int32")].reshape((x.shape[0],1,x.shape[1],Words.shape[1]))                                  
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/tensor/var.py", line 359, in flatten
    return theano.tensor.basic.flatten(self, ndim)
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/tensor/basic.py", line 5300, in flatten
    x_reshaped, *filter(lambda i: broadcastable[i], range(ndim)))
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/tensor/basic.py", line 4092, in addbroadcast
    rval = Rebroadcast(*[(axis, True) for axis in axes])(x)
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/gof/op.py", line 615, in __call__
    node = self.make_node(*inputs, **kwargs)
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/compile/ops.py", line 710, in make_node
    if self.axis.keys() or (x.ndim <= max(self.axis.keys())):
ValueError: max() arg is an empty sequence
